includes/wporg-meeting-posttype.php

```diff
@@ -86,6 +93,11 @@ public function meeting_set_next_meeting( $posts, $query ) {
 			}
 		});
 
+		// Only sort if every post is a meeting CPT
+		if ( $meeting_count !== count( $posts ) ) {
```

Comment:
With the above this would be: `if ( (array) $query->post_type !== [ 'meeting' ] )`